tests/Unit/API/Site/Controllers/Assets/AssetSuggestionsControllerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\Site\Controllers\Ads;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Ads\AssetSuggestionsService;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\Assets\AssetSuggestionsController;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Framework\RESTControllerUnitTest;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * Class AssetSuggestionsControllerTest
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\Site\Controllers\Ads
+ *
+ * @property RESTServer                         $rest_server
+ * @property AssetSuggestionsService|MockObject $asset_suggestion
+ */
+class AssetSuggestionsControllerTest extends RESTControllerUnitTest {
+
+	protected const ROUTE_FINAL_URL_SUGGESTIONS = '/wc/gla/assets/final-url/suggestions';
+	protected const TEST_FINAL_URL_SUGGESTIONS  = [
+		[
+			'id'    => 1,
+			'type'  => 'post',
+			'title' => 'Beanie with Logo',
+			'url'   => 'http://localhost:8082/shop/clothing/accessories/beanie-with-logo/',
+		],
+		[
+			'id'    => 2,
+			'type'  => 'term',
+			'title' => 'Clothing',
+			'url'   => 'http://localhost:8082/product-category/clothing/',
+		],
+
+	];
+	protected const TEST_NO_FINAL_URL_SUGGESTIONS = [];
+
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->asset_suggestion = $this->createMock( AssetSuggestionsService::class );
```

Comment:
💅  Minor

This could be changed as well 

`$this->asset_suggestions_service`